cmd/zoekt-webserver/main.go

```diff
@@ -643,12 +646,14 @@ func newGRPCServer(logger sglog.Logger, streamer zoekt.Streamer, additionalOpts
 
 	opts := []grpc.ServerOption{
 		grpc.ChainStreamInterceptor(
+			propagator.StreamServerPropagator(tenant.Propagator{}),
```

Comment:
In Sourcegraph services we also have a middleware that wraps incoming requests with goroutine labels so in profiles we see which tenant causes which go routines - would this be helpful here as well?